datasette_write_ui/table.ts

```diff
@@ -103,20 +100,52 @@ class RowIcon {
 
   hide() {
     this.menu.style.display = "none";
+    this.menu.isShown = false;
   }
   show() {
+    // Close other menus
+    Array.from(document.querySelectorAll(".datasette-write-ui-menu")).forEach(
+      (menu) => {
+        menu.style.display = "none";
+        menu.isShown = false;
+      }
+    );
+    const wrapper = document.body;
     this.menu.style.display = "block";
+    this.menu.isShown = true;
+    absolutelyReparent(this.menu, wrapper);
```

Comment:
This is the key trick: the element was being cut off because it was absolutely positioned inside an element that had `overflow:x-auto` on it.

The easiest way I could find to fix this was to switch the menu to being absolutely positioned within `document.body` itself - calculating the offsets, re-parenting it and positioning it that way.